packages/predicates/src/plugins/ownership-predicate.ts

```diff
@@ -0,0 +1,104 @@
+import BigNum = require('bn.js')
+
+import {
+  getOverlappingRange,
+  PredicatePlugin,
+  StateUpdate,
+  SyncManager,
+  Transaction,
+} from '@pigi/core'
+
+export class OwnershipPredicatePlugin implements PredicatePlugin {
+  private readonly syncManager: SyncManager
+
+  public constructor(syncManager: SyncManager) {
+    this.syncManager = syncManager
+  }
+
+  public async executeStateTransition(
+    previousStateUpdate: StateUpdate,
+    transaction: Transaction,
+    witness: string
+  ): Promise<StateUpdate> {
+    await this.validateStateTransition(
+      previousStateUpdate,
+      transaction,
+      witness
+    )
+
+    const range = getOverlappingRange(
```

Comment:
Isn't range supposed to be transaction.start to transaction.end? See [docs](http://spec.plasma.group/en/latest/src/07-predicates/simple-ownership.html#state-transitions)

Reply:
Good call -- I was just worried about checking that they overlap. Also need to make sure that the `transaction.range` is a subset of the `previousStateUpdate.range`